tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -7,3 +7,15 @@ tests:
     arch_exclude: riscv32 nios2 posix
     platform_exclude: qemu_x86_coverage qemu_cortex_m0
     tags: kernel userspace
+  kernel.timer.tickless.stm32:
+    filter: dt_compat_enabled("stm32,lptim")
```

Comment:
This compatible does not actually exists (LPTIM1 compat is "st,stm32-timers").
But indeed a dedicated lptim compatible is required so we don't mix with other general purpose timers

Reply:
adding a new compatible  = "st,stm32-lptim  especially for lptim, which is itself 'timers" compatible (9416b3d7d1ef91a2c0a3fc826e3b40ea6404871f)